apps/frontend/src/components/ui/textarea.tsx

```
@@ -11,22 +15,27 @@
       <textarea
         ref={ref}
         data-slot="textarea"
-        rows={rows} 
+        rows={rows}
```

Biome (1.9.4)

[error] This JSX property is assigned multiple times.

This attribute is assigned again here.


(lint/suspicious/noDuplicateJsxProps)